plugins/inputs/mcrouter/mcrouter.go

```diff
@@ -0,0 +1,224 @@
+package mcrouter
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+// Mcrouter is a mcrouter plugin
+type Mcrouter struct {
+	Servers     []string
+	UnixSockets []string
+}
+
+var sampleConfig = `
+  ## An array of address to gather stats about. Specify an ip on hostname
+  ## with optional port. ie localhost, 10.0.0.1:11211, etc.
+  servers = ["localhost:11211"]
+  # unix_sockets = ["/var/run/mcrouter.sock"]
+`
+
+var defaultTimeout = 5 * time.Second
```

Comment:
Let's make this configurable and have it apply to all servers.

Reply:
I followed the zookeeper example here